src/content/getusermedia/resolution/js/main.js

```diff
@@ -103,7 +103,8 @@ video.onplay = function() {
 function getMedia(constraints) {
   if (stream) {
     video.src = null;
-    stream.stop();
+    stream.getAudioTracks().forEach(function(track) { track.stop(); });
```

Comment:
@fippo, maybe create an issue in the adapter repo for that? Then bump adapter.js in a separate PR.